### PR TITLE
TW 464 Collection Runner refresh

### PR DESCRIPTION
**Pull request by HeatherStenson**
- Updated screenshots
- Numbered procedures
- General edits + ran Vale

### CODE REVIEW

**Review by mark-anthony-postman — APPROVED** [review]
Nice refresh, LGTM! 🦄
Made some minor suggestions you can use as you wish 🙂

File: src/pages/docs/running-collections/intro-to-collection-runs.md [hunk 1]
```diff
@@ -30,87 +30,89 @@ warning: false
...
 
-<img alt="Run history" src="https://assets.postman.com/postman-docs/run-history-v8.jpg" width="350px"/>
+    > You can also select <img alt="Runner icon" src="https://assets.postman.com/postman-docs/icon-runner-v9.jpg#icon" width="16px"> __Runner__ from the Postman footer and drag a collection over from __Collections__ or __History__ in the sidebar.
 
-By default your requests will run in the sequence they're listed in the collection. If you need to change the order of execution, select a request and drag it to move it to its new order. You can also deselect an individual request by unchecking its box.
+1. Before you start a collection run, you can choose optional configuration parameters:
 
-[![Runner request order](https://assets.postman.com/postman-docs/runner-order-v8.jpg)](https://assets.postman.com/postman-docs/runner-order-v8.jpg)
+    * **Iterations** - The number of iterations for your collection run. You can also run collections multiple times against different data sets to [build workflows](/docs/running-collections/building-workflows/).
```
[the remark this answers is not in the document]
Comment (HeatherStenson) [reply src/pages/docs/running-collections/intro-to-collection-runs.md:60]: We don't have an official rule yet - this is up for discussion in Cycle 3, in fact! I'm partial to using a colon but will be fine with whatever we choose as long as we're consistent. I'll hold off on making any changes until we have the final ruling. 🔨

File: src/pages/docs/running-collections/intro-to-collection-runs.md [hunk 2]
```diff
@@ -30,87 +30,89 @@ warning: false
...
+To return to the overview of the run, select __View Results__.
 
-[![Runner console](https://assets.postman.com/postman-docs/run-console-v8.jpg)](https://assets.postman.com/postman-docs/run-console-v8.jpg)
+### Viewing test results
 
-To export test results, select **Export Results**. To view details about the run, select __View Summary__.
+If your collection has tests, you will see whether each request passed or failed. You can also filter on each using the __Passed__ and __Failed__ tabs at the top. If any tests in a request script fail during the collection run, the whole request fails.
 
-[![Runner result summary](https://assets.postman.com/postman-docs/run-overview-v8.jpg)](https://assets.postman.com/postman-docs/run-overview-v8.jpg)
+> If you check __Save responses__ when you start a collection run, you will see the responses in the run results.
```
[the remark this answers is not in the document]
Comment (HeatherStenson) [reply src/pages/docs/running-collections/intro-to-collection-runs.md:95]: AHHHHHHH how did this sneak through! 🪦

File: src/pages/docs/running-collections/intro-to-collection-runs.md [hunk 3]
```diff
@@ -30,87 +30,89 @@ warning: false
...
-The summary also lists a column for each iteration of your collection run so that you can see test output for the whole run at a glance.
+### Viewing iterations
 
-To return to the overview of the run, select __View Results__.
+If you set an iteration number for the collection run, you can see the overview each iteration by selecting one from the list on the right.
 
 ## Sharing collection runs
 
-You can share the run results for a collection, by exporting them from the Collection Runner so that other people can analyze them.
+You can share collection run results with other people by exporting them from the Collection Runner.
```
[the remark this answers is not in the document]
Comment (HeatherStenson) [reply src/pages/docs/running-collections/intro-to-collection-runs.md:103]: Lol, good call 😆

**Pull request merged**